front/lib/api/assistant/actions/process.ts

```diff
@@ -267,11 +266,7 @@ export class ProcessConfigurationServerRunner extends BaseActionConfigurationSer
     // Handle data sources list and parents/tags filtering.
     config.DATASOURCE.data_sources = actionConfiguration.dataSources.map(
       (d) => ({
-        workspace_id:
-          isDevelopment() && !apiConfig.getDevelopmentDustAppsWorkspaceId()
-            ? PRODUCTION_DUST_WORKSPACE_ID
-            : d.workspaceId,
```

Comment:
Using process/retrieval on a local setup connected on prod core is already not working : the fallback on dust workspace cannot be used anymore since we rely dataSourceView id instead of dataSource name - so removing it.